README.md

```diff
@@ -303,6 +303,109 @@ The credit card charge result may contains `redirect_url` for 3DS authentication
 For full example on Credit Card 3DS transaction refer to:
 - [Flask App examples](/examples/flask_app) that implement Snap & Core Api
 
+### 2.2.D Subscription API
+You can see some Subscription API examples [here](examples/subscription), [Subscription API Docs](https://api-docs.midtrans.com/#subscription-api)
+
+Subscription API only support for `Credit Card` and `Gopay`
+
+#### Subscription API for Credit Card
+
+To use subscription API for credit card, you should first obtain the 1-click saved token, [refer to this docs.](https://docs.midtrans.com/en/core-api/advanced-features?id=recurring-transaction-with-subscriptions-api) 
+
+You will receive `saved_token_id` as part of the response when the initial card payment is accepted (will also available in the HTTP notification's JSON), [refer to this docs.](https://docs.midtrans.com/en/core-api/advanced-features?id=sample-3ds-authenticate-json-response-for-the-first-transaction)
+
+```python
+# Create Subscription API instance
+core_api = midtransclient.CoreApi(
+    is_production=False,
+    server_key='YOUR_SERVER_KEY',
+    client_key='YOUR_CLIENT_KEY'
+)
+# Prepare parameter
+param = {
+    "name": "SUBSCRIPTION-STARTER-1",
+    "amount": "100000",
+    "currency": "IDR",
+    "payment_type": "credit_card",
+    "token": "436502qFfqfAQKScMtPRPdZDOaeg7199",
+    "schedule": {
+      "interval": 1,
+      "interval_unit": "month",
+      "max_interval": 3,
+      "start_time": "2021-10-01 07:25:01 +0700"
+    },
+    "metadata": {
+      "description": "Recurring payment for STARTER 1"
+    },
+    "customer_details": {
+      "first_name": "John A",
+      "last_name": "Doe A",
+      "email": "johndoe@email.com",
+      "phone": "+62812345678"
+    }
+}
+create_subscription_response = core_api.create_subscription(param)
+
+subscription_id_response = create_subscription_response['id']
+# get subscription by subscription_id
+get_subscription_response = core_api.get_subscription(subscription_id_response)
+
+# disable subscription by subscription_id
+disable_subscription_response = core_api.disable_subscription(subscription_id_response)
+
+# enable subscription by subscription_id
+enable_subscription_response = core_api.enable_subscription(subscription_id_response)
+
+# update subscription by subscription_id
+update_param = {
+    "name": "SUBSCRIPTION-STARTER-1-UPDATE",
+    "amount": "100000",
+    "currency": "IDR",
+    "token": "436502qFfqfAQKScMtPRPdZDOaeg7199",
+    "schedule": {
+      "interval": 1
+}
+update_subscription_response = core_api.update_subscription(subscription_id_response, update_param)
+```
+
+#### Subscription API for Gopay
+
+To use subscription API for gopay, you should first link your customer gopay account with gopay tokenization API, [refer to this section](#22e-tokenization-api)
+
+You will receive gopay payment token using get_payment_account API call 
+
+Check [Tokenization examples](/examples/tokenization)
```

Comment:
Why is this sentence inconsistent to other samples mentioning example? like
```
You can see some Subscription API examples [here](examples/subscription)
```
When writing docs consistency is important. Please note, be more careful & learn for next time.